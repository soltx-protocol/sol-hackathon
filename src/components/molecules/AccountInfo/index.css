.account-info {
	width: 100%;

	& > .header {
		font-weight: 700;
		font-size: 24px;
		line-height: 24px;
		color: #d9dcff;
	}

	& > .account {
		font-weight: 400;
		font-size: 14px;
		line-height: 20px;
		color: #d9dcff;
	}

	& > .coin {
		margin-top: 46px;
		font-weight: 400;
		font-size: 14px;
		line-height: 20px;
		color: #d9dcff;
	}
}
