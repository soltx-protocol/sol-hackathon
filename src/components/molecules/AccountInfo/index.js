import React from 'react';
import classnames from 'classnames';

import { useAccount } from 'models/account';
import styles from './index.css';

const AccountInfo = ({ className }) => {
	const [{ account }] = useAccount();

	return (
		<div className={classnames(styles.accountInfo, className)}>
			<div className={styles.header}>Individual Account </div>
			<div className={styles.account}>{account === '' ? '-' : account}</div>
			<div className={styles.coin}>Asset:</div>
		</div>
	);
};

export default AccountInfo;
