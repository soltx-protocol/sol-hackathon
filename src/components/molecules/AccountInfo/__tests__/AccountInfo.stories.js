import React from 'react';
import { storiesOf } from '@storybook/react';

import AccountInfo from 'components/molecules/AccountInfo';

const stories = storiesOf('molecules/AccountInfo', module);

stories.add('__interactive', () => <AccountInfo />, {
	redux: {
		data: {
			account: {
				address: '',
				pools: [],
			},
			routing: {
				pathname: '/',
			},
		},
	},
});
