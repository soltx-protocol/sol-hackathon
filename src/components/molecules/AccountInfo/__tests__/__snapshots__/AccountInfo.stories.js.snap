// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="accountInfo"
  >
    <div
      class="header"
    >
      Individual Account 
    </div>
    <div
      class="account"
    >
      -
    </div>
    <div
      class="coin"
    >
      Asset:
    </div>
  </div>
</DocumentFragment>
`;
